package.json

```diff
@@ -26,7 +26,10 @@
     "url": "https://github.com/mulesoft/api-console/issues"
   },
   "homepage": "https://github.com/mulesoft/api-console",
-  "dependencies": {},
+  "dependencies": {
+    "grunt-karma": "^0.10.1",
```

Comment:
Why adding karma and grunt-karma into the dependency since they are already part in the devDependencies and should be there?